bin/pycbc_optimize_snr

```diff
@@ -351,14 +355,24 @@ assert maxchirp < 250.
 # Establish minimum eta: find the most asymmetric mass point
 # nb function name is 'mass2' but it doesn't enforce m2 < m1!
 maxm1 = cv._mass2_from_mchirp_mass1(maxchirp, MIN_CPT_MASS)
-mineta = max(cv.eta_from_mass1_mass2(maxm1, MIN_CPT_MASS), 0.01)
+mineta = max(cv.eta_from_mass1_mass2(maxm1, MIN_CPT_MASS), MIN_ETA)
+
+# Use max chirp and min eta to find the upper bound on mass2
+maxm2 = cv.mass2_from_mchirp_eta(maxchirp, MAX_ETA)
+
+# Astrophysical spin bounds dependent on system masses - https://inspirehep.net/literature/1124003
```

Comment:
I don't see a reasoning for limiting the spin to +-0.9 in the BBH case in that reference?

Reply:
I'm aware that this was the limit set before, but the comment implies that it will be explained for both cases in the reference

Reply:
@spxiwh was there ever a reasoning for limiting the spin to +-0.9 for BBH?